### PR TITLE
Added cheat sheet for the node.js basics

### PR DESCRIPTION
**Pull request by NoSQLKnowHow**

### CODE REVIEW

**Comment by NoSQLKnowHow** [comment]
Getting this out and then will add more example API calls soon.

File: docs/develop/guides/cheat-sheets/momento-cache-nodejs-cheat-sheet.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    .catch((e: Error) => {
+      console.error(`Uncaught exception while running example: ${e.message}`);
+      throw e;
+    });
+  ```
+
+  </TabItem>
+  <TabItem value="nodejs" label="Node.js" default>
+    Coming soon.
+  </TabItem>
```
Comment (eaddingtonwhite) [inline docs/develop/guides/cheat-sheets/momento-cache-nodejs-cheat-sheet.md:73]: would nodjs look very different? just wondering if can just have single tab instead seperate typescript and nodejs ones. I feel like would look identical. Fine to wait can tackle that in later PR's though

File: docs/develop/guides/cheat-sheets/momento-cache-nodejs-cheat-sheet.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+  <TabItem value="js" label="JavaScript" default>
+    Coming soon.
+  </TabItem>
+</Tabs>
+
+## Running the code
+If you download the [entire file](https://github.com/momentohq/client-sdk-nodejs/blob/main/examples/basic.ts), then run this command to execute the code.
+
+```cli
+npx ts-node basic.ts
```
Comment (eaddingtonwhite) [inline docs/develop/guides/cheat-sheets/momento-cache-nodejs-cheat-sheet.md:175]: if we were to add plain js example would this look that same? Or would we need tabs here. Maybe another reason just to have TS example only?

**Review by eaddingtonwhite — APPROVED** [review]
One nit on if we actually need to have js example or not. Id be fine just having TS.

**Pull request merged**